Fulfill pessimistic variant of work for 14.03.2019

marcinogo opened the pull request:
I make basic setup for OX project, create flow diagram and crc cards for applicaion.

marcin-ogorzalek-EPAM committed:
Write progress estimates for 14.03.2019

marcin-ogorzalek-EPAM committed:
Add dependency for TestNG

marcin-ogorzalek-EPAM committed:
Update section AboutOX

marcin-ogorzalek-EPAM committed:
Create .gitattributes

marcin-ogorzalek-EPAM committed:
Create .mailmap file

marcin-ogorzalek-EPAM committed:
Update estymaty.md for 14.03.2019

marcin-ogorzalek-EPAM committed:
Change to proper description of project

marcin-ogorzalek-EPAM committed:
Create flow diagrams for application

marcin-ogorzalek-EPAM committed:
Create CRC cards for application

WojTechM commented:
Niespójności w diagramach przepływu i kartach CRC. Zdjęcia kartek papieru wątpliwej rozdzielczości nie ułatwiają zrozumienia. Rozważ przepisanie w formie elektronicznej

marcinogo committed:
Redraw flowchart for the project

Due to low quality, long time of loading jpg file live on github and
mistakes flowcharts had been redrawn using Dia appliaction.

marcinogo committed:
Add new process to setup flowchart

marcinogo committed:
Redraw CRC cards

Due to low qulity and mistakes CRC radds has been redrawn.
